packages/esm-service-queues-app/src/queue-table/queue-table.component.tsx

```diff
@@ -101,15 +104,13 @@ export const QueueTableByStatus: React.FC = () => {
         selectedIndex={currentStatusIndex}
         onChange={({ selectedIndex }) => setCurrentStatusUuid(allowedStatuses[selectedIndex].uuid)}
         className={styles.tabs}>
-        <TabList style={{ paddingLeft: '1rem' }} aria-label="Outpatient tabs" contained>
+        <TabList style={{ paddingLeft: '1rem' }} aria-label={t('queueStatus', 'Queue Status')} contained>
           {allowedStatuses?.map((s) => <Tab key={s?.uuid}>{s?.display}</Tab>)}
         </TabList>
         <TabPanels>
           {allowedStatuses?.map((s) => (
             <TabPanel key={s?.uuid} style={{ padding: 0 }}>
-              <QueueTable
-                visitedQueueEntriesByLocationAndQueueAndStatus={visitedQueueEntriesByLocationAndQueueAndStatus}
-              />
+              <QueueTable visitQueueEntriesByLocationAndQueueAndStatus={visitQueueEntriesByLocationAndQueueAndStatus} />
```

Comment:
The entries we are passing in here have nothing to do with `ByLocation`, do they?  Filtering by `Queue` and `Status` is enough.  I don't believe we need to do anything with Location?

Reply:
Right now, I'm doing `useUnmappedVisitQueueEntries(currentLocationUuid)`, which is only getting `visitQueueEntries` from the current location (hard coded to user's session location right now). I haven't done this yet, but I plan to have the list of queues dropdown (from `useQueues()`) be limited by current location as well, and hook up the location dropdown currently defined in the queue app's home page to control this location.

Reply:
The whole "mapped"  vs "unmapped" visit queue entries thing is a bit confusing to me, to be honest.  Do we need to have both of these?  I understand the value in having a nicer object to work with than the REST response, but this doesn't feel exactly right.  What is done in other esms that have user-friendly objects to work with and map to/from rest representations?

Regarding the other point, this should really depend on the nature of the overall page that is using this component.  I would not expect variable names (or assumptions) like this within a "queue-table" component.  If there is a page called "queues-for-location", then it would make sense to have variable names in that page that have names like that.  But in a generic queue table component - no.

Reply:
I intend to get rid of the `MappedVisitQueueEntry` when this new table is more feature complete and can replace the existing table. The existing table uses `MappedVisitQueueEntry`, so for now supporting both the unmapped and mapped versions is meant to be an un-invasive way to reuse old code while we try to migrate. I can add some comment about that. (@mogoodrich is also getting rid of the similar `MappedAppointment` type in his other PR)